tests/test_subprocesses.py

```diff
@@ -63,3 +64,19 @@ def terminate(signum, frame):
 
         process.terminate()
         assert await process.wait() == 2
+
+
+async def test_process_cwd(tmp_path):
+    """Test that `cwd` is successfully passed to the subprocess implementation"""
+    cmd = [sys.executable, "-c", "import os; print(os.getcwd())"]
+    result = await run_process(cmd, cwd=str(tmp_path))
```

Comment:
Why force it to str? It's declared to support `Path` objects, right?

Reply:
The ability to pass `cwd` as a path object was added to `subprocess.Popen` on Windows in Python 3.7.  However I tried it on a Windows box on python 3.6 and it seems to have worked, so I'll change it.